compiler/control/OMROptions.hpp

```diff
@@ -190,7 +190,7 @@ enum TR_CompilationOptions
    TR_EnableJITServerHeuristics           = 0x00020000 + 3,
    TR_SoftFailOnAssume                    = 0x00040000 + 3,
    TR_DisableNewBlockOrdering             = 0x00080000 + 3,
-   TR_DisableZNext                        = 0x00100000 + 3,
+   TR_Disablez17                          = 0x00100000 + 3,
```

Comment:
Please keep `TR_DisableZNEXT` and add new entry for `TR_DisableZ17` - Also make sure you capitalize Z.